src/sentry/rules/conditions/level.py

```diff
@@ -58,7 +58,7 @@ def passes(self, event, state, **kwargs):
             return False
 
         desired_level = int(desired_level)
-        level = int(event.level)
+        level = int(event.group.level)
```

Comment:
I didn't even realize that we didn't support reading the event level (from anything other than the `data`, at least.)